From a548e40e5e67b147b96d003aae7a76b790919928 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Thu, 16 Apr 2020 09:01:13 +0800
Subject: [PATCH] ARM: dts: rk3288: adjust pvtm-voltage-sel table

Current pvtm-voltage-sel table is not stable for some chips, reported by
some clients.

Change-Id: If2f7d8cd6a077922334c8e6ba4b01f82e27048d7
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk3288.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index f304841c9a0b..42ddf9bf1a1e 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -110,10 +110,10 @@
 			3               31
 		>;
 		rockchip,pvtm-voltage-sel = <
-			0        14300   0
-			14301    15000   1
-			15001    16000   2
-			16001    99999   3
+			0        15300   0
+			15301    16000   1
+			16001    17000   2
+			17001    99999   3
 		>;
 		rockchip,pvtm-freq = <408000>;
 		rockchip,pvtm-volt = <1000000>;
-- 
2.35.3

